README.md

```diff
@@ -0,0 +1,107 @@
+Run celestia-app chain
+---
+
+## Pre-Requisites 
```

Comment:
I don't think these pre-requisites are necessary to list here, only that a celestia node needs a `trusted-peer` and `trusted header hash`. Maybe we can have these prerequisite instructions in their respective repositories?

Reply:
Make sense. Re-working.
I'll also add celestia-node part down below after celestia-app's last chapter